arts-university-bournemouth.csl

```diff
@@ -183,8 +183,8 @@
           </group>
         </if>
         <else>
-          <group delimiter=", ">
-            <group delimiter=", ">
+          <group delimiter=". ">
+            <group delimiter=". ">
```

Comment:
@Dbelk, results in `(D’Arcus. 2006).` and `(D’Arcus. 2006. p. 3).`. The use of periods here is very atypical. s this correct?